### PR TITLE
Add --no-sandbox option to desktop files

### LINKED ISSUE #1703: FATAL:setuid_sandbox_host.cc(157)
**What version of this package are you using?**
0.21 deb
**What operating system, Node.js, and npm version?**
kali linux, v12.10.0 , 6.10.3
**What happened?**
imagen
**What did you expect to happen?**
which will start the program normally
**Are you willing to submit a pull request to fix this bug?**
no

### PR DESCRIPTION
**Pull request by hicom150**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[x] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

Add `--no-sandbox` option to launch scripts (Desktop files) to temporary fix #1703 issue.

**Is there anything you'd like reviewers to focus on?**

Check on different platforms. I've checked successfully in Ubuntu 18.04.3 LTS and Debian 8 Jessie

### CODE REVIEW

**Comment by feross** [comment]
Did you confirm that it didn't work before this change and then it worked with the change?

File: static/linux/webtorrent-desktop.ejs [hunk 1]
```diff
@@ -5,7 +5,7 @@ Name=<%= productName %>
 <% if (genericName) { %>GenericName=<%= genericName %><% } %>
 <% if (description) { %>Comment=<%= description %><% } %>
 Icon=<%= name %>
-<% if (name) { %>Exec=<%= name %> %U<% } %>
+<% if (name) { %>Exec=<%= name %> --no-sandbox %U<% } %>
```
Comment (feross) [inline static/linux/webtorrent-desktop.ejs]: Can you add a comment after each line with `--no-sandbox` to note that this is a temporary hack? Something like:

```
<!-- HACK: --no-sandbox fixes an Electron 6 bug. See: #1703 -->
```

**Comment by hicom150** [comment]
> Did you confirm that it didn't work before this change and then it worked with the change?

Yes, I've been able to reproduce the whole issue and the fix following the steps described [here](https://github.com/webtorrent/webtorrent-desktop/issues/1703#issuecomment-533295632).

File: static/linux/webtorrent-desktop.ejs (hunk 1 above)
Comment (hicom150) [reply static/linux/webtorrent-desktop.ejs]: Sure! 👍

**Pull request merged**